tests/rptest/services/redpanda.py

```diff
@@ -346,30 +346,34 @@ class SISettings:
     ABS_AZURITE_ACCOUNT = "devstoreaccount1"
     ABS_AZURITE_KEY = "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
 
-    def __init__(self,
-                 test_context,
-                 *,
-                 log_segment_size: int = 16 * 1000000,
-                 cloud_storage_credentials_source: str = 'config_file',
-                 cloud_storage_access_key: str = 'panda-user',
-                 cloud_storage_secret_key: str = 'panda-secret',
-                 cloud_storage_region: str = 'panda-region',
-                 cloud_storage_api_endpoint: str = 'minio-s3',
-                 cloud_storage_api_endpoint_port: int = 9000,
-                 cloud_storage_cache_size: int = 160 * 1000000,
-                 cloud_storage_enable_remote_read: bool = True,
-                 cloud_storage_enable_remote_write: bool = True,
-                 cloud_storage_max_connections: Optional[int] = None,
-                 cloud_storage_disable_tls: bool = True,
-                 cloud_storage_segment_max_upload_interval_sec: Optional[
-                     int] = None,
-                 cloud_storage_manifest_max_upload_interval_sec: Optional[
-                     int] = None,
-                 cloud_storage_readreplica_manifest_sync_timeout_ms: Optional[
-                     int] = None,
-                 bypass_bucket_creation: bool = False,
-                 cloud_storage_housekeeping_interval_ms: Optional[int] = None,
-                 fast_uploads=False):
+    def __init__(
+            self,
+            test_context,
+            *,
+            log_segment_size: int = 16 * 1000000,
+            cloud_storage_credentials_source: str = 'config_file',
+            cloud_storage_access_key: str = 'panda-user',
+            cloud_storage_secret_key: str = 'panda-secret',
+            cloud_storage_region: str = 'panda-region',
+            cloud_storage_api_endpoint: str = 'minio-s3',
+            cloud_storage_api_endpoint_port: int = 9000,
+            cloud_storage_cache_size: int = 160 * 1000000,
+            cloud_storage_enable_remote_read: bool = True,
+            cloud_storage_enable_remote_write: bool = True,
+            cloud_storage_max_connections: Optional[int] = None,
+            cloud_storage_disable_tls: bool = True,
+            cloud_storage_segment_max_upload_interval_sec: Optional[
+                int] = None,
+            cloud_storage_manifest_max_upload_interval_sec: Optional[
+                int] = None,
+            cloud_storage_readreplica_manifest_sync_timeout_ms: Optional[
+                int] = None,
+            bypass_bucket_creation: bool = False,
+            cloud_storage_housekeeping_interval_ms: Optional[int] = None,
+            # TODO: update spillover size to 4KiB to exercise behaviour
```

Comment:
nit: stray TODO?

Reply:
Nope, by default spillover is turned off in tests. The comment is saying to turn it on once the read path is aware of it.